client/__tests__/e2e/config.js

```diff
@@ -6,3 +6,5 @@ export const appUrlBase =
 export const DATASET = "pbmc3k";
 export const isDev = jestEnv === ENV_DEFAULT.DEV;
 export const isDebug = jestEnv === ENV_DEFAULT.DEBUG;
+export const TEST_EMAIL = "user@example.com";
```

Comment:
Ah `user@example.com` is only available in staging, so we'll need to create the same account in prod Auth0 tenant!